frontend/types/Autonolas.ts

```diff
@@ -23,3 +23,15 @@ export type StakingContractInfo = {
   /** OLAS cost of staking */
   minStakingDeposit: number;
 };
+
+export type OtherStakingContractInfo = {
+  /** annual percentage yield */
+  apy: number;
+  /** amount of OLAS required to stake */
+  stakeRequired: number;
+  /** rewards per work period */
+  rewardsPerWorkPeriod: number;
+};
+
+export type StakingContractRecord = Partial<StakingContractInfo> &
```

Comment:
please extend `StakingContractInfo` instead, making the fields optional throughout or just using Partial<>🙏